### DISCUSSION TITLE
Is eye tracking from the tutorial working?

### DISCUSSION THREAD

**Discussion by IsayDBS** [jspsych/jsPsych#3086] [Q&A]
I try to run the example given by the documentation with the version 7.0.0, but it doesn't even asks me to turn on the camera, the page stays on white, any idea why? Or any other example I can check out?

**Comment by ayugioh2003** [comment]
Do you mean the examples on this page?

- https://www.jspsych.org/7.3/overview/eye-tracking/#example
- https://www.jspsych.org/7.3/demos/eye-tracking-with-webgazer.html

I have fixed some bugs this page and merged them into the main branch. I think the next version of the official documentation should be fixed.

- https://github.com/jspsych/jsPsych/pull/3150

If you just want to see the effect of the example, you can check out my codesandbox.

- https://codesandbox.io/s/jspsych-eye-tracking-example-2992dh
